packages/config/config/devices/0x0460/qnsw-001P16.json

```
@@ -29,31 +28,58 @@
 			"maxNodes": 9
 		}
 	},
+
 	"paramInformation": [
 		{
 			"#": "1",
 			"label": "Sw1 Switch Type",
-			"description": "0",
+			"description": "This parameter defines how the Device should treat the switch (which type) connected to the SW (SW1) terminal.",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 2,
 			"defaultValue": 2,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "mono-stable switch type (push button)",
+					"value": 0
+				},
+				{
+					"label": "toggle switch (contact closed - ON / contact opened - OFF)",
+					"value": 1
+				},
+				{
+					"label": "toggle switch (Device changes status when switch changes status)",
+					"value": 2
+				}
+			]
 		},
 		{
 			"#": "17",
 			"label": "Restore State of O1 After Power Failure",
```

We try to be consistent across device configs from different manufacturers. So this should be called
```suggestion
			"label": "O1: State After Power Failure",
```

---

Also, these parameters are so common that we have specific [templates](https://zwave-js.github.io/node-zwave-js/#/config-files/using-templates?id=using-templates) for them to be imported, in this case:
https://github.com/zwave-js/node-zwave-js/blob/5e89385694154acc3e58dc787bf89debcbd84b16/packages/config/config/devices/templates/master_template.json#L501

This entire parameter would just become:
```jsonc
	{
		"$import": "~/templates/master_template.json#state_after_power_failure_prev_off",
		"label": "O1: State After Power Failure",
	}
```